massa-protocol-worker/src/tests/operations_scenarios.rs

```diff
@@ -280,6 +279,10 @@ async fn test_protocol_propagates_operations_only_to_nodes_that_dont_know_about_
     .await;
 }
 
+/*
+
+TODO: the following test might be obsolete with the new algorithm
+
 #[tokio::test]
 #[serial]
 async fn test_protocol_propagates_operations_only_to_nodes_that_dont_know_about_it_block_integration(
```

Comment:
Here there is some tests that a probably obsoletes.

We can just check if the protocol call correctly the network when it should without testing the full content of the response.

We can test that in followup. Likewise, we also probably don't need to start all that mocks to test f the operation propagation work correctly.

In follow up, with the new testes after the main also implementation, we should remove that part.

Reply:
To avoid a lost of time I would like to add inside:

```suggestion
/*

TODO: the following test might be obsolete with the new algorithm, if you see hat comment in a month, please delete me

#[tokio::test]
#[serial]
async fn test_protocol_propagates_operations_only_to_nodes_that_dont_know_about_it_block_integration(
```

Reply:
outdated